kyaml/fn/framework/resourcelist.go

```diff
@@ -0,0 +1,187 @@
+// Copyright 2019 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package framework
```

Comment:
Framework overview section is the most important section that function authors will go through. And it can be greatly simplified. I think the current content seems to be outdated because it is using `kpt run` commands instead of `kpt fn eval`.

I think that is coming from `doc.go` or something.